docs/02-connect/04-eon_block_explorer.md

```diff
@@ -0,0 +1,584 @@
+- **Gas Limit** - The maximum amount of total gas consumed for all transactions in one block.
+- **Gas Used** - The total amount of gas used by all transactions in this block.
+
+## Transactions  
+The Transactions section includes:
+
+- Validated 
+- [Forward Transfer](#forward-transfer) 
+- [Fee Payments](#fee-payment)
+- Pending
+- Verified Contracts
+
+### Validated Transactions
+Transaction validation indicates that a transaction conforms to a specific protocol requirement to deem it as valid. Once the transaction is considered valid it is added to the distributed ledger.
+ 
+The **Validated Transaction** feature provides key information about the transaction.
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-validated.png").default} alt="validated-transaction" width="900" height="700" />
+</p>
+
+
+Key information includes:
+
+- **Transaction Status** - Success or Error: (Awaiting internal transaction for reason)
+- **Transaction ID (hash)** - The unique identifier that validates the transaction.
+- **Address** - The wallet address initiating the transaction. 
+- **Transaction Fee** - The amount charged to process the transaction.
+- **Contract Address** - Click the address to Address Detail page.
+- **(Receiving) Address** - Click the address receiving asset to Address Detail page.
+- **Block Number** - The number assigned to the block.
+- **Elapsed timer for the block** - A timer that displays the elapsed time since block creation period.
+
+### Pending Transactions
+Currently this feature is not enabled.
+
+### Verified Contracts 
+A verified contract shares its source code with the public, including the compiler setting. The verification process compares the generated bytecode with the one deployed on-chain.
+
+The Verified Contracts feature provides key information about the contract.
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-verified-contract.png").default} alt="verified-contracts" width="900" height="700" />
+</p>
+
+
+Key information includes:
+
+- **Total Number of Contracts in the Last 24 hours** - The total number of verified contracts in the last 24 hours.
+- **Address** - The contract address. 
+- **Balance** - The total amount of tZEN associated with the smart contract address.
+- **Transaction (Txns)** - The number of transactions associated with the verified contract.
+- **Compiler** - The name of the compiler type.
+- **Version** - The version number of the compiler type.
+- **Optimization** - A green check mark indicates that optimization is applied. A red cross indicates that optimization was not applied.
+- **Construction Args** - A green check mark indicates that construction arguments were used. A red cross indicates that construction arguments were not used.
+- **Verified** - A timer that displays the elapsed time since the contract was verified.
+- **Market Cap** - The value of the token if it is bought and sold with fiat currency.
+
+# Tokens Menu

```

Comment:
This doesn't seem to be showing up as a section in the sidebar navigation menu. I would expect this to be a section and `All` and `tZEN` to be sub-sections of this

Reply:
made correction